### ISSUE TITLE
tf.layers.batch_normalization报错

### ISSUE THREAD

**Issue by simon1024** [alibaba/x-deeplearning#185]
在使用tf.layers.batch_normalization的时候，报如下错误：
<pre>
[WARNING]: no gradient found for var:batch_normalization/moving_mean under scope: , maybe not used?
[WARNING]: no gradient found for var:batch_normalization/moving_variance under scope: , maybe not used?
ERROR:root:ErrorCode[3], ErrorMsg[tf session run failed, errormsg:Input 0 of node batch_normalization/AssignMovingAvg was passed float from _arg_batch_normalization/moving_mean_0_31:0 incompatible with expected float_ref.
Check Status [tf_runner_.Run(tf_inputs, output_op_names, &results)] at [/xdl/xdl/xdl/core/backend/tf/backend_op.cc]Compute@90]
</pre>

请问，xdl里支持batch_normalization么，如果不支持，可有替代方案？
